samples/Meeting/src/app/views/MeetingScreen.tsx

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { GroupCallLocator, TeamsMeetingLinkLocator } from '@azure/communication-calling';
+import { CommunicationUserIdentifier } from '@azure/communication-common';
+import { MeetingAdapterState, MeetingComposite, toFlatCommunicationIdentifier } from '@azure/communication-react';
+import { Spinner } from '@fluentui/react';
+import React, { useEffect, useRef, useState } from 'react';
+import { useSwitchableFluentTheme } from '../theming/SwitchableFluentThemeProvider';
+import { createAutoRefreshingCredential } from '../utils/credential';
+import MobileDetect from 'mobile-detect';
+import { createAzureCommunicationMeetingAdapter } from '@azure/communication-react';
+import { MeetingAdapter } from '@internal/react-composites';
+
+const detectMobileSession = (): boolean => !!new MobileDetect(window.navigator.userAgent).mobile();
+
+export interface MeetingScreenProps {
+  token: string;
+  userId: CommunicationUserIdentifier;
+  callLocator: GroupCallLocator | TeamsMeetingLinkLocator;
+  displayName: string;
+  webAppTitle: string;
+  onMeetingEnded: () => void;
+  endpoint: string;
+  threadId: string;
+}
+
+export const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
+  const { token, userId, callLocator, displayName, webAppTitle, onMeetingEnded, threadId, endpoint } = props;
+  const [adapter, setAdapter] = useState<MeetingAdapter>();
+  const callIdRef = useRef<string>();
+  const adapterRef = useRef<MeetingAdapter>();
+  const { currentTheme, currentRtl } = useSwitchableFluentTheme();
+  const [isMobileSession, setIsMobileSession] = useState<boolean>(detectMobileSession());
+
+  useEffect(() => {
+    if (!callIdRef.current) {
+      return;
+    }
+    console.log(`Call Id: ${callIdRef.current}`);
+  }, [callIdRef.current]);
+
+  // Whenever the sample is changed from desktop -> mobile using the emulator, make sure we update the formFactor.
+  useEffect(() => {
+    const updateIsMobile = (): void => setIsMobileSession(detectMobileSession());
+    window.addEventListener('resize', updateIsMobile);
+    return () => window.removeEventListener('resize', updateIsMobile);
+  }, []);
+
+  useEffect(() => {
+    (async () => {
+      if (!userId || !displayName || !callLocator || !threadId || !token || !endpoint) {
+        return;
+      }
+      const adapter = await createAzureCommunicationMeetingAdapter({
+        userId,
+        displayName,
+        credential: createAutoRefreshingCredential(toFlatCommunicationIdentifier(userId), token),
+        callLocator: callLocator,
+        endpoint,
+        chatThreadId: threadId
+      });
+      adapter.on('meetingEnded', () => {
+        onMeetingEnded();
+      });
+      // adapter.on('error', (e) => {
+      // Error is already acted upon by the Call composite, but the surrounding application could
```

Comment:
should we just remove this comment?

Reply:
according to James this implementation of the meeting adapter is coming so I just commented it out for now but am happy to remove it in the short term.

Reply:
I've added a comment to the work item so feel free to remove this: https://skype.visualstudio.com/SPOOL/_backlogs/backlog/ACS%20Web%20UI/Features/?workitem=2590727